src/modules-load/modules-load.c

```diff
@@ -17,12 +17,30 @@
 #include "proc-cmdline.h"
 #include "string-util.h"
 #include "strv.h"
+#include <pthread.h>
```

Comment:
Please include system header first, then include ours. Also, please sort the headers:
```diff
diff --git a/src/modules-load/modules-load.c b/src/modules-load/modules-load.c
index c1f818320f..76fd73e546 100644
--- a/src/modules-load/modules-load.c
+++ b/src/modules-load/modules-load.c
@@ -3,6 +3,7 @@
 #include <errno.h>
 #include <getopt.h>
 #include <limits.h>
+#include <pthread.h>
 #include <sys/stat.h>
 
 #include "build.h"
```